osidb/tests/test_snippet.py

```diff
@@ -49,38 +49,22 @@ def test_create_flaw_from_snippet(
         assert flaw.workflow_state == WorkflowModel.WorkflowState.NEW
 
         flaw_cvss = flaw.cvss_scores.all().first()
-        assert flaw_cvss.issuer == FlawCVSS.CVSSIssuer.NIST
-        assert flaw_cvss.score == 8.1
-        assert flaw_cvss.vector == "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H"
-        assert flaw_cvss.version == FlawCVSS.CVSSVersion.VERSION3
+        for i in ["issuer", "score", "vector", "version"]:
+            assert getattr(flaw_cvss, i) == snippet.content["cvss_scores"][0][i]
 
         flaw_ref = flaw.references.all().first()
-        assert flaw_ref.type == FlawReference.FlawReferenceType.SOURCE
-        assert flaw_ref.url == "https://nvd.nist.gov/vuln/detail/CVE-2023-0001"
+        for i in ["type", "url"]:
+            assert getattr(flaw_ref, i) == snippet.content["references"][0][i]
 
         flaw_snippet = flaw.snippets.all().first()
         assert flaw_snippet == snippet
-        assert flaw_snippet.content == content
-        assert flaw_snippet.external_id == snippet.external_id
-        assert flaw_snippet.source == snippet.source
+        for i in ["content", "external_id", "source"]:
+            assert getattr(flaw_snippet, i) == getattr(snippet, i)
```

Comment:
I do not think that these shortcuts are a good idea for the tests. It makes it harder to debug when something goes wrong. A simple `assert X == Y` without any cycle, getter, indexing, etc. is maybe a verbose but easy to investigate structure. I do not have too strong preference however so leaving it up to you.